src/libraries/System.Text.Encodings.Web/tests/SR.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    internal static partial class SR
```

Comment:
How come we're using resource strings in a test project?

Reply:
This test project, in the netstandard build, has some tests for Rune and it runs these tests by linking the Rune.cs and ThrowHelper.cs files from the src project. ThrowHelper.cs uses SR so the test project needs to define it to make everything compile. It's not used for anything outside of this.